packages/documentation-framework/components/themeSelector/themeSelector.js

```diff
@@ -103,6 +103,7 @@ export const ThemeSelector = ({ id }) => {
       )}
       shouldFocusToggleOnSelect
       onOpenChangeKeys={['Escape']}
+      popperProps={{ position: 'right', preventOverflow: true, enableFlip: true }}
```

Comment:
Should we default to right when there's probably room to show it on the left?

Reply:
Oh sorry I was specifically referring to the one in the masthead, which is the most prominent one. On org it's left aligned currently, but right aligned in this PR:

Org
<img width="1432" height="466" alt="Screenshot 2025-09-16 at 2 56 31 PM" src="https://github.com/user-attachments/assets/8682d862-03f5-4622-b289-3e5c52683ccb" />

This PR
<img width="1762" height="600" alt="Screenshot 2025-09-16 at 2 56 54 PM" src="https://github.com/user-attachments/assets/ff875434-83a1-4013-ab8c-51f9b1e89531" />

Reply:
Right! The release dropdown is right positioned already so I made it match. It looks a little funky to me otherwise, but I ultimately don't really care.

<img width="339" height="333" alt="Screenshot 2025-09-16 at 4 10 47 PM" src="https://github.com/user-attachments/assets/61abe4e9-2772-45a7-b9ba-a8f51bfc0a5c" />

Reply:
Oh gotcha! I didn't realize you were comparing it to the version switcher. Checked with @lboehling and she's good with this change and prefers right aligned things here in general (search bar slides out from the right, too) 🚀